RELEASE-NOTES.txt

```diff
@@ -9,6 +9,7 @@
 - [*] Products: Downloadable products now accept file types other than images from WordPress media library. [https://github.com/woocommerce/woocommerce-ios/pull/11356]
 - [*] Payments menu: restored the ability to search for Payments in device Spotlight. [https://github.com/woocommerce/woocommerce-ios/pull/11343]
 - [*] Payments menu: show the selected payment gateway when there's more than one to choose from [https://github.com/woocommerce/woocommerce-ios/pull/11345]
+- [*] Edit Products: category list is now searchable. [https://github.com/woocommerce/woocommerce-ios/pull/11380]
```

Comment:
🔔 We can move this to 16.7

Reply:
Moved in 51bbf2ff429c5f67a918190f25f0c55a9915efc0 ✅